### PR TITLE
Fix issue #10 : numeric task_id is now well processed

### PR DESCRIPTION
**Pull request by fbaligand**

### CODE REVIEW

**Comment by fbaligand** [comment]
@purbon 
As promised, the pull request to add version 0.1.4 description to CHANGELOG.md
Can you review merge it ?

Fabien

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
    instead of using Thread.raise on the plugins' threads. Ref: https://github.com/elastic/logstash/pull/3895
  - Dependency on logstash-core update to 2.0
 
+# v 0.1.4
```
Comment (purbon) [inline CHANGELOG.md]: In master, as we introduced some stop breaking changes, not much affecting filters, but we raised all versions to be ```2.x```, we should align with this versions. Check the ```.gemspec``` for more details.

Comment (fbaligand) [inline CHANGELOG.md]: @purbon 
I understand.
But he situation is little bit complicated :
- in v1 compatible version, the fix is available from 0.1.4
- in v2 compatible version, the fix is available from 2.0.2

So what should we do in CHANGELOG.md ?
Add the fix in both 0.1.4 and 2.0.2 descriptions ?

Comment (purbon) [inline CHANGELOG.md]: in v1 branch, changelog should show the fix for 0.1.4 while for v2 the fix goes in for the next available version. makes sense?

**Comment by fbaligand** [comment]
This is an option...
But the strange thing is that in master branch, README.md includes versions 0.1.2 and 0.1.3. 
So why not 0.1.4 ?

**Comment by purbon** [comment]
I think you mean the ```CHANGELOG.md```, not the ```README.md```, right? well the answer to me is easy, this change, the one that involved this was after we raised master to be 2.x, so the natural version for master is in the 2.x level, otherwise it would be like injecting something in between, right?

**Comment by fbaligand** [comment]
I speak about CHANGELOG.md, sorry for the mistake :)

Okay for what you suggest.
That makes sense.
=> I change CHANGELOG.md in that direction.

**Comment by fbaligand** [comment]
@purbon 
That's done :)

**Comment by fbaligand** [comment]
Hi @purbon, 

Can you merge the pull request ?

**Comment by fbaligand** [comment]
Thus, is it possible to have write rights on issues ?

Otherwise, could you do the following on issue #5 :
- add "bug", "v0.1.4" and "v2.0.2" tags
- change title to : "catch and log exceptions during code call execution"
- close issue

Thanks,
Fabien

**Comment by purbon** [comment]
@fbaligand let me see what I can do, working on this now.

**Comment by fbaligand** [comment]
@purbon thanks.
But firstable, can you merge this pull request ? 
I need this to sync my fork with this repo and then start to work on issue #10

**Comment by fbaligand** [comment]
@purbon 
Since this "changelog update" pull request takes so long to be merged, I finally turned this pull request for fix issue #10.

Could you review it ?

**Comment by suyograo** [comment]
@fbaligand for the indentation fix, can you also submit a seperate PR to core-api-v1 branch?

**Comment by fbaligand** [comment]
@suyograo I plan to create a PR on core-api-v1 branch when this PR will be reviewed and merged.
Like that, I could merge master branch to core-api-v1 branch.

**Comment by purbon** [comment]
@fbaligand @suyograo we can always rebase and backport to the core-api-v1 if possible and necessary.

@fbaligand took a while, because I've been afk traveling. This is why we should soon setup proper mantainers for plugins, with commit rights, et all..... not so far away.

**Comment by purbon** [comment]
please jenkins, test this.

**Comment by purbon** [comment]
@fbaligand I see "strange" commits in this PR history, like the ones that add the badge, not sure why they are there.

**Comment by fbaligand** [comment]
Hi @purbon,
for the strange history, it's my fault : I had done a "git pull" thinking I will get only update on my origin/master, but I got also upstream/master changes, including your "badge add".

=> Sorry for that, that was a bad idea :(

**Comment by purbon** [comment]
@fbaligand ok, no problem. Can you probably redo the PR? this LGTM so we can merge, but I think makes sense to have a clean history, right?

**Comment by fbaligand** [comment]
By the way, I love your new jenkins test automation launch ! :)

**Comment by purbon** [comment]
@fbaligand my please, this is one of the things in the direction we're working to have tons of tests :+1:

**Comment by fbaligand** [comment]
OK to make another PR for a cleaner history.
I could do it tonight.

If you want to go in this direction, feel free to close this PR.

**Comment by purbon** [comment]
Thanks @fbaligand, I will merge very fast as it makes sense this PR is LGTM. closing this one in favor a feature PR with a clean history.

**Comment by fbaligand** [comment]
Concerning backport to branch "core-api-v1", I clearly think it is really useful : to have fix for logstash 1.5 and have a cleaner doc in logstash reference documentation.

I think that not everybody will migrate to logstash 2.0 immediatly :)

**Comment by purbon** [comment]
ok, makes sense.

**Comment by fbaligand** [comment]
@purbon 
For "core-api-v1" branch, do you make the backport ? Or Do I create a new PR for that ?

**Comment by purbon** [comment]
@fbaligand up to you, If the PR is clean enough is easy for me to merge booths in one shot. But I guess documentation wise might be harder.

**Comment by fbaligand** [comment]
After thinking about that, the backport is not straightforward : all updates on master branch are not to backport on branch "core-api-v1". So I will make a specific PR for that.

**Comment by purbon** [comment]
+1 thanks @fbaligand

**Pull request closed without merging**